src/Nethermind/Nethermind.Synchronization/Peers/SyncPeersReport.cs

```diff
@@ -69,7 +70,58 @@ public void WriteAllocatedReport()
                     return;
                 }
 
-                _logger.Info(MakeReportForPeer(OrderedPeers.Where(p => (p.AllocatedContexts & AllocationContexts.All) != AllocationContexts.None), $"Allocated sync peers {_currentInitializedPeerCount}({_peerPool.PeerCount})/{_peerPool.PeerMaxCount}"));
+                var header = $"Allocated sync peers {_currentInitializedPeerCount}({_peerPool.PeerCount})/{_peerPool.PeerMaxCount}";
+                if (_logger.IsDebug)
+                {
+                    _logger.Debug(MakeReportForPeer(OrderedPeers.Where(p => (p.AllocatedContexts & AllocationContexts.All) != AllocationContexts.None), header));
+                }
+                else
+                {
+                    _logger.Info(MakeSummaryReportForPeers(_peerPool.InitializedPeers, header));
+                }
+            }
+        }
+
+        internal string? MakeSummaryReportForPeers(IEnumerable<PeerInfo> peers, string header)
+        {
+            lock (_writeLock)
+            {
+                IEnumerable<IGrouping<string, PeerInfo>> peerGroups = peers.GroupBy(peerInfo => ParseClientInfo(peerInfo.SyncPeer.ClientId));
```

Comment:
Can we avoid parsing? We have `ClientType`.

Reply:
Changed